scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -558,8 +558,18 @@ def lvm_exist(self):
 
 def cluster_up(self):
     cmd = 'hctl status'
-    op = subprocess.run(["hctl", "status"])
+    op = subprocess.run(list(cmd.split(' ')))
     if op.returncode == 0:
         return True
     else:
         return False
+
```

Comment:
A new function is defined but where is it called ?

Reply:
Updated file was missed to update.
Updated c4914b1